docs/source/api/v3/acme_accounts.rst

```diff
@@ -0,0 +1,197 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-acme-accounts:
+
+*****************
+``acme/accounts``
+*****************
+
+``GET``
+=======
+Gets information for all ACME accounts.
+
+:Auth. Required: Yes
+:Roles Required: "admin"
+:Response Type:  Array
+
+Request Structure
+-----------------
+No parameters available
+
+
+Response Structure
+------------------
+:email:       The account information as defined in :ref:`cdn.conf`.
+:private_key: The private key connected to the ACME account
```

Comment:
Per API guidelines, encoded property names should use camelCase (`privateKey` not `private_key`)